src/mlpack/tests/serialization.hpp

```diff
@@ -22,7 +22,6 @@
 #include <mlpack/core.hpp>
 
 #include <boost/test/unit_test.hpp>
-#include "test_tools.hpp"
```

Comment:
It's probably a good idea to include this anyway, in case `CheckMatrices()` is ever used in this test.  The file is very short anyway.  (For what it's worth, it looks like some of the reverse compatibility code in `test_tools.hpp` is very out of date and can be removed---the minimum Boost version is greater than 1.36, so that bit can go if you would like to remove it.)